docs/components/configMaps.md

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.configMaps`
+
+This grade provides the configuration maps for the gamepad in the form of component's model data.
+
+## Using this grade
+
+This can be used by passing it to the component's `gradeNames` option.
+
+``` javascript
```

Comment:
You can fairly safely assume that people working in the Fluid ecosystem don't need this simple of an example.   IMO this would be a good place to demonstrate actually mixing it with a functional grade.